bindings/go/serde.go

```diff
@@ -0,0 +1,226 @@
+package veriblock
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"errors"
+	"io"
+	"math"
+	"reflect"
+	"unsafe"
+)
+
+// Parse - parses hex string to byte array.
+func Parse(src string) []byte {
```

Comment:
I do not think you need to have this func, because you already have this `hex.DecodeString` func.
It can fail if input string has non-hex chars